data.json

```diff
@@ -89,6 +89,15 @@
     "dateAdded": "2020-07-21T14:42:00.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "Camunda",
+    "difficulty": "medium",
+    "description": "Submit 2 pull requests to any of their repositories to get a t-shirt",
```

Comment:
And stickers (`Participants who complete the Camunda challenge will earn this limited edition Camunda x Hacktoberfest 2020 t-shirt as well as these nifty stickers.`)